src/Common.re

```diff
@@ -0,0 +1,9 @@
+  [@bs.deriving {jsConverter: newType}]
+  type bounds = {
+    x: int,
+    y: int,
+    width: int,
+    height: int,
+  };
+
+  [@bs.send] external getBoundsJs: 'a => abs_bounds = "getBounds";
```

Comment:
Wouldn't it make sense here to create a helper function that does the conversion for you?

So add 

`let getBounds = (a) => getBoundsJs(a)->boundsFromJs`